cmd/data-scanner.go

```diff
@@ -1109,21 +1109,21 @@ func (i *scannerItem) applyActions(ctx context.Context, o ObjectLayer, oi Object
 	return size
 }
 
-func evalActionFromLifecycle(ctx context.Context, lc lifecycle.Lifecycle, lr lock.Retention, obj ObjectInfo) (action lifecycle.Action) {
-	action = lc.ComputeAction(obj.ToLifecycleOpts())
+func evalActionFromLifecycle(ctx context.Context, lc lifecycle.Lifecycle, lr lock.Retention, obj ObjectInfo) (action lifecycle.Action, sc string) {
```

Comment:
It seems to me like `sc` could easily be part of the `lifecycle.Action` as `DestStorageClass`, or do we have limited options there?

Reply:
We can definitely add that @klauspost

Reply:
Would be nice. Otherwise LGTM